views/guide.lavender

```diff
@@ -89,8 +89,9 @@ block content
           h2#github via Github      
           p
             | Add the lavender submodule with 
-            code git submodule add git@github.com:golavender/lavender.git (folder to 
-            | clone to) and include it with 
+            code git submodule add git@github.com:golavender/lavender.git (folder to
+            code clone to)
```

Comment:
Cool tidbit here for you @jennEDVT , if it's a small change like this, you can just do a quick edit in GitHub. Go to the 'Files Changed' tab, and click on the pencil to the left of the file name (it should pop up saying something about file changes).
Do a quick fix, commit the changes, and it's done. Just don't go crazy and do lines and lines of code.